liberty-managed/JakartaEE9_README.md

```diff
@@ -59,6 +71,11 @@ To enable Arquillian Liberty Managed in your project, add the following to your
 		<version>2.0.0</version>
 		<scope>test</scope>
 	</dependency>
+	<dependency>
+		<groupId>io.openliberty.arquillian</groupId>
+		<artifactId>arquillian-liberty-rest-protocol-jakarta</artifactId>
+		<version>2.0.0</version>
```

Comment:
I think this will be `2.1.0` right? Might as well change the one above this to match at `2.1.0`.

Reply:
I have updated this in the PR.  Things will be updated again when there is a new Alpha release of Arquillian.

Should I also update the support user feature to be 2.1 as well since everything else is moving to 2.1?

Reply:
I'm not sure what the criteria is for updating the version of the support feature. We did not update it for the minor fix releases (2.0.1, 2.0.2 etc). It is not published to Maven Central right? I think the versioning for the support feature is completely separate from the versioning of the Maven artifacts.

Reply:
@kathrynkodama Can you chime in here on versioning of the support feature?

Reply:
The support feature artifact is published to maven central and will be of version 2.1.0, but the liberty user feature is still 2.0.  You thought about an update to the user feature version when I changed it to use JMX instead of servlet, so it could be appropriate to change it.

Reply:
It is confusing. The support feature artifact is published, but is the support feature itself? The version of the support feature is in this file: https://github.com/OpenLiberty/liberty-arquillian/blob/main/liberty-support-feature/src/feature/resources/arquillian-liberty-support.mf

Reply:
Both are published.  See https://repo.maven.apache.org/maven2/io/openliberty/arquillian/arquillian-liberty-support-jakarta/2.0.2/arquillian-liberty-support-jakarta-2.0.2-feature.zip for the feature being published.

Reply:
k..I'm fine with changing it to 2.1 if we think it makes sense. Some documentation would need updating as well.

Reply:
+1 I think if we are bumping all of the versions to `2.1.0`, we should update to `arquillian-support-jakarta-feature-2.1` to match

Reply:
PR is updated with the change.